### PR TITLE
[475] Update canonical meta tag and other things 😅

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
- Naming

### Changes proposed in this pull request
- Update to canonical meta tag to the production domain
- Update cookie session name
- Update remaining boilerplate stuff to actual app name

####Cookie name before and after
Screenshot 2020-11-18 at 21 26 07

Screenshot 2020-11-18 at 21 26 12

#### Canonical meta tag before and after
Screenshot 2020-11-18 at 21 27 53
Screenshot 2020-11-18 at 21 28 02

### Guidance to review
🚢

### CODE REVIEW

**Comment by dankmitchell** [comment]
[update canonical link](https://trello.com/c/oreldEqt/475-update-canonical-link)

File: config/initializers/canonical_rails.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   # This is the main host, not just the TLD, omit slashes and protocol. If you have more than one, pick the one you want to rank in search results.
 
-  config.host = "www.education.gov.uk"
+  config.host = "www.register-trainee-teachers.eduation.gov.uk"
```
Comment (defong) [inline config/initializers/canonical_rails.rb:10]:
```suggestion
  config.host = "www.register-trainee-teachers.education.gov.uk"
```

**Review by defong — CHANGES_REQUESTED** [review]

**Review by defong — APPROVED** [review]
Keeping it fun

File: config/application.rb [hunk 2]
```diff
@@ -42,5 +42,7 @@ class Application < Rails::Application
 
     config.middleware.use Rack::Deflater
     config.active_job.queue_adapter = :sidekiq
+
+    config.session_store :cookie_store, key: '_register_trainee_teachers_session'
```
Comment (defong) [inline config/application.rb:46]:
```suggestion
    config.session_store :cookie_store, key: "_register_trainee_teachers_session"
```

**Review by avinhurry — APPROVED** [review]

**Pull request merged**